### PR TITLE
Fixed conflicting ref_id:s between 4.2.1 and the hh.bib in the last master

### PR DESCRIPTION
**Pull request by d97hah**

### CODE REVIEW

File: languoids/tree/indo1319/indo1320/indo1321/indo1323/oriy1254/gaud1237/gaud1238/uncl1515/kurm1243/md.ini [hunk 1]
```diff
@@ -5,7 +5,7 @@ hid = kfv
 level = language
 iso639-3 = kfv
 latitude = 26.9175
-longitude = 87.3205
+longitude = 87.32050000000001
```
Comment (xrotwang) [inline languoids/tree/indo1319/indo1320/indo1321/indo1323/oriy1254/gaud1237/gaud1238/uncl1515/kurm1243/md.ini:8]: This one - and the `latitude = 1.9144999999999999` above - keep coming back. Are coordinates refreshed automatically on your end?

**Comment by xflr6** [comment]
Thanks! SGTM. For the record remaining changed `glottolog_ref_id`s between 341cf6d1d19ee80a6b084ee75978e839a45abe54
and ac37e6eda511fac6887bd61243a646660dd4137d (the last two look like a move):

```
changed
d:Thomas:Ibo                             577906 90455
g:Avrorin:Nanajskogo                     11706 568497
s:Avrorin:Nanajskomu                     568497 42583
s:Smith:Nupe                             577373 58964
t:ArfstenVanselow:Frasch                 472744 578134
t:ArfstenVanselow:Solring                578134 472742
```

**Comment by xrotwang** [comment]
So @d97hah should we merge?

**Comment by d97hah** [comment]
Yes

Pada tanggal Kam, 14 Mei 2020 pukul 09.16 Robert Forkel <
notifications@github.com> menulis:

**Pull request merged**